reference/interbit-ui-components/CONTRIBUTING.adoc

```diff
@@ -0,0 +1,205 @@
+ */
+export default class LinkBar extends Component {
+  static propTypes = {
+    /**
+     * A short title describing the call-to-action for LinkBar
+     */
+    title: PropTypes.string.isRequired,
+    /**
+     * A description of the call-to-action for LinkBar
+     */
+    content: PropTypes.string.isRequired,
+    /**
+     * An image that provides context for the call-to-action
+     */
+    image: PropTypes.string,
+    /**
+     * A URL, relative path, or mailto link. E.g. `https://interbit.io`,  `/pages/foo`, or `mailto:foo@interbit.io`
+     */
+    to: PropTypes.string,
+    /**
+     * CSS classes to be applied to the component
+     */
+    className: PropTypes.string,
+    /**
+     * A flag. Set to `true` if the `to` property is a `mailto:` link
+     */
+    isMailto: PropTypes.bool,
+    /**
+     * A function that is fired when a user clicks on the LinkBar
+     */
+    clickHandler: PropTypes.func
+  }
+...
+}
+```
+
+Each UI Kit component should be accompanied by an example file,
+`MyComponent.md`. link:https://react-styleguidist.js.org/docs/documenting.html[React Styleguidist]
+will render the components and JSDoc comments in a style guide.
+
+The example file should include:
+- An example of the rendered component.
+- An example of each variant of the component. For example, we might apply
+`className="blue"` to a button component to render a blue button.
+
+The following is `LinkBar`'s example `md` file:
+
+````
+Full-sized examples. To see how the component behaves responsively, change your
+viewport width.
+
+```jsx
+<div>
+  <LinkBar to="http://google.com" image="placeholder-md.png" title="Default LinkBar" content="First were dominion third sea very years upon the. Over also moving also so let." />
+  <LinkBar to="http://google.com" className="blue" image="placeholder-md.png" title="Blue LinkBar" content="Apply the 'blue' class to the component. First were dominion third sea very years upon the." />
+  <LinkBar to="http://google.com" className="dotted" image="placeholder-md.png" title="Dotted LinkBar" content="Apply the 'dotted' class to the component. First were dominion third sea very years upon the." />
+</div>
+```
+````
+
+The style guide can be run locally from the `interbit-ui-components` directory
+by executing the following commands:
+```sh
+npm run styleguide:build
+npm run styleguide
+```
+
+== Modifying existing components
+
+=== Tests
+
+Run the test watcher from the `interbit-ui-components` directory with:
+```
+npm run test
+```
+
+Fix any tests that were broken after modifying the component, and add tests for
+any new behavior.
+
+=== Documentation
+
+Update the JSDoc code comments and the React Styleguidist examples to reflect
+the changes made to the component. Document any breaking changes.
+
+
+== Submitting your work
+
+Create a branch on the Interbit repo and create a
```

Comment:
External developers cannot create a branch. They have to fork the repository and create a pull request from their fork/branch to our repo/master.

Reply:
👍